user-guide/Basic_Functionality/Trending/Setting_the_default_sliding_window_size_for_real-time_trending.md

```diff
@@ -13,6 +13,9 @@ To configure the default size:
 
 1. Go to System Center \> *System settings* > *time to live*
 
+   > [!NOTE]
+   > From DataMiner 10.3.0 [CU15]/10.4.0 [CU3]/10.4.6 onwards<!--RN 39173-->, the *System settings* > *time to live* tab in System Center is no longer available if you are using a [DaaS system](xref:Creating_a_DMS_in_the_cloud), as these settings are automatically configured as part of DaaS.
```

Comment:
...in System Center is temporary no longer available...

Reply:
Will the TTL settings be available again in a future DataMiner version?

Reply:
there are plans to do so. However, it is not written in stone that it will be enabled again in the future.
Basically, this is disabled because it would not work at all with current code and there will be an attempt to let it work again.

Reply:
In that case, I wouldn't add the word "temporarily", so as not to create expectations that may not be fulfilled. However, if in the future the settings become available again, we can change the note to something along the lines of:

> [!NOTE]
> From DataMiner 10.3.0 [CU15]/10.4.0 [CU3]/10.4.6 up to DataMiner 10.4.12/10.5.0, the *System settings* > *time to live* tab in System Center are temporarily unavailable if you are using a [DaaS system](xref:Creating_a_DMS_in_the_cloud).

Reply:
Let's keep it as is, maybe the best option :)